command-list.txt

```diff
@@ -166,6 +166,7 @@ git-show-index                          plumbinginterrogators
 git-show-ref                            plumbinginterrogators
 git-sh-i18n                             purehelpers
 git-sh-setup                            purehelpers
+git-sparse-checkout                     mainporcelain           worktree
```

Comment:
I did not add git-gvfs-helper to this list.  I don't see entries for things like git-remote-curl nor -https, so I think that is correct.

Reply:
This is mostly for the help text.

Reply:
I think that `git-remote-curl` might be missing from `command-list.txt` because it is compiled only conditionally... (you can opt out from compiling it by setting `NO_CURL=1` in the `make` invocation.)